packages/documents-list-widget/src/widgets/documents-list-widget/app.tsx

```diff
@@ -1,8 +1,13 @@
 import React, { memo } from 'react';
 import DocumentsListWidget from './components/documents-list-widget';
+import {initTranslations} from './init';
+
+// Import .po files as translations using our custom Vite plugin
+const translationsModules: Record<string, { default: Record<string, string>; locale: string }> = import.meta.glob('../../../translations/*.po', { eager: true })
```

Comment:
@andrey-skl do we load translations for all languages here?

Reply:
Yes. For widgets they contains like dozens of strings, so (and it was always like that for widgets) I don't think it makes sense to complicate infra by lazy-loading